test/e2e/build_kubectl_buildkit_test.go

```diff
@@ -104,3 +109,16 @@ spec:
 		t.Fatalf("Expected >>>%s<<< to match >>>%s<<<", out, expectedOut)
 	}
 }
+
+func createBuilder(t *testing.T) {
+	var stdoutBuf, stderrBuf bytes.Buffer
+
+	cmd := exec.Command("kubectl", "buildkit", "create")
+	cmd.Stdout = &stdoutBuf
+	cmd.Stderr = &stderrBuf
+
+	err := cmd.Run()
+	if err != nil {
+		t.Fatalf(fmt.Sprintf("error: %s\n", err))
```

Comment:
no need for fmt.Sprintf since we are using Fatalf

Reply:
fixed